File: Interview 62. LCOF/lastRemaining.java
class lastRemaining {
    public int solution(int n, int m) {
    	int p = 0;
    	for (int i = 2; i < n + 1; i++) {
    		p = (p + m) % i;
    	}
    	return p;
    }

    public static void main(String[] argv) {
    	int n = 5;
    	int m = 3;
    	lastRemaining obj = new lastRemaining();
    	int res = obj.solution(n, m);
    	System.out.println(res);
    }
}

File: LCOF 62./lastRemaining.py
class lastRemaining:
    def solution(self, n: int, m: int) -> int:
        p = 0
        for i in range(2,n+1):
        	p = (p + m) % i
        return p



obj = lastRemaining()
n = 5
m = 3
res = obj.solution(n, m)
print(res)
